src/Cassandra.IntegrationTests/Core/SchemaMetadataTests.cs

```diff
@@ -34,11 +34,8 @@ public SchemaMetadataTests() :
             base(() =>
             {
                 string[] cassandraYaml = null;
-                if (TestClusterManager.CheckCassandraVersion(true, new Version(5, 0), Comparison.GreaterThanOrEqualsTo))
-                {
-                    cassandraYaml = new[] { "materialized_views_enabled: true" };
-                }
-                else if (TestClusterManager.CheckCassandraVersion(true, new Version(4, 0), Comparison.GreaterThanOrEqualsTo))
+                if (TestClusterManager.CheckCassandraVersion(true, new Version(4, 0), Comparison.GreaterThanOrEqualsTo) ||
+                    TestClusterManager.IsHcd)
```

Comment:
Why did you remove `cassandraYaml = new[] { "materialized_views_enabled: true" };` for `5.0+`?

Reply:
Same for user defined functions

Reply:
Because we fix them here: https://github.com/datastax/csharp-driver/blob/7eaff90f96b786e725e33821fcef19460d112071/src/Cassandra.IntegrationTests/TestClusterManagement/CcmBridge.cs#L340-L373